### PR TITLE
Add repository application.properties file

### LINKED ISSUE #24: Add application propeties in CommonRepository
Make it connect using hibernate and the SQL language and declare database credentials

### PR DESCRIPTION
**Pull request by dottore1**
closes #24 
## Description
Provide details about your Pull Request and what it adds, fixes or
changes.

## Checklist:
- [x] My code follows the code style of this project.
- [x] I have added tests to cover my changes, if necessary.
- [ ] All tests pass (existing and potential new).
- [ ] Sonar check has passed
- [ ] No codesmells or bugs

## Other Relevant Information
Provide any other important details below.

### CODE REVIEW

File: BrewMesModulesParent/common_repository/src/main/resources/application.properties [hunk 1]
```diff
@@ -1 +1,3 @@
-T
+spring.application.name=Repository
+spring.data.mongodb.uri=#add uri here!
```
Comment (Kaszz) [inline BrewMesModulesParent/common_repository/src/main/resources/application.properties:2]: mongodb+srv://admin:admin@brewcluster.qp2xo.mongodb.net

Comment (Kaszz) [inline BrewMesModulesParent/common_repository/src/main/resources/application.properties:2]:
```suggestion
spring.data.mongodb.uri=mongodb+srv://admin:admin@brewcluster.qp2xo.mongodb.net
```

**Review by Wylardt — APPROVED** [review]

**Review by Lawst96 — APPROVED** [review]
I hate it

**Review by Kechr — APPROVED** [review]

**Review by Teyson — APPROVED** [review]

**Review by Kaszz — APPROVED** [review]
🐸

**Pull request merged**